tests/test_value_utils.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+import magma as m
+from bit_vector import BitVector
+from fault.array import Array
+from fault.value_utils import make_value
+from fault.value import AnyValue, UnknownValue
+
+
+class Foo(m.Circuit):
+    IO = ["a", m.In(m.Bit),
+          "b", m.In(m.Bits(8)),
+          "c", m.In(m.Array(12, m.Bit)),
+          "d", m.In(m.Array(16, m.Array(20, m.Bit)))]
+
+
+def test_all():
```

Comment:
This test could be structured better, but just wanted to get in all the test functionality so we're sure nothing is broken.

Reply:
pytest parametrization might be useful here (see https://docs.pytest.org/en/latest/parametrize.html)